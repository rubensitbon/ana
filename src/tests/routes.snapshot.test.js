// @flow
import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Routes from '../routes';

describe('<Routes>', () => {
  it('should render correctly', () => {
    const tree = shallow(<Routes />);
    expect(toJSON(tree)).toMatchSnapshot();
  });
});
