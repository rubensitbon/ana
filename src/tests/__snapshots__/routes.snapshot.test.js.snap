// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Routes> should render correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/avatar"
  />
</Switch>
`;
